spec/connection_spec.cr

```diff
@@ -0,0 +1,11 @@
+require "./spec_helper"
+
+describe Avram::Connection do
+  it "displays a helpful error when failing to connect" do
+    conn = Avram::Connection.new("postgres://root:root@localhost:5432/tacoman")
+    message = Regex.new("Failed to connect to databse 'tacoman' with username root.")
```

Comment:
I believe you can pass the string and it will do a substring match. I'm not 100% sure though but worth a try

Reply:
I tried that first and it fails. Seems the string does exact match?